packages/cms-cli/bin/cli.js

```diff
@@ -35,9 +35,6 @@ const argv = yargs
   .middleware([setLogLevel])
   .exitProcess(false)
   .fail((msg, err, _yargs) => {
-    // Preserve stack trace.
-    if (err) throw err;
```

Comment:
Left another suggestion related to this over in https://github.com/HubSpot/hubspot-cms-tools/pull/299/files#r480221294 after doing some testing and further reviewing.

Reply:
Changed to process.exit(1)